rebar.config

```diff
@@ -1,6 +1,8 @@
 {minimum_otp_vsn, "21.0"}.
 
-{deps, [{getopt, "1.0.1"}]}.
+{deps, [ {getopt, "1.0.1"}
+       , {typerefl, {git, "https://github.com/k32/typerefl", {tag, "0.3.0"}}}
```

Comment:
`0.3.1` supports IP address types from the box

Reply:
applied